tests/Sdk/Unit/Extension/Workflow/InteractionAnswerBasedTransitionResolverTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Unit\Extension\Workflow;
+
+use Codeception\Test\Unit;
+use SprykerSdk\Sdk\Extension\Workflow\InteractionAnswerBasedTransitionResolver;
+use SprykerSdk\SdkContracts\Entity\ContextInterface;
+use SprykerSdk\SdkContracts\ValueReceiver\ValueReceiverInterface;
+
+class InteractionAnswerBasedTransitionResolverTest extends Unit
```

Comment:
Please provide test that checks if exception is thrown if it is a valid case.

Reply:
done